tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -267,6 +267,9 @@ Standard Section: "Comments" Comment*
 ```none
   Comment       = Length Bytes LongInt      // Raw comment's bytes encoded as UTF-8, followed by the comment's coordinates.
 ```
+
+Standard Section: "Attributes" UTF8*
```

Comment:
We should specify here what are the possible attributes. They have semantic meaning, not just a collection of strings, so they should be specified.